ldms/src/sampler/spank/slurm_notifier.c

```diff
@@ -427,9 +429,15 @@ void add_client(struct client_list *cl, const char *spec)
 void setup_clients(int argc, char *argv[], struct client_list *cl)
 {
 	const char *timeout = NULL;
+	const char *dbg = NULL;
 	int rc;
 
 	for (rc = 0; rc < argc; rc++) {
+		if (0 == strncasecmp(argv[rc], "debug", 5)) {
+			dbg = get_arg_value(argv[rc]);
+			if (dbg && dbg[0] == '1')
+				user_debug = 1;
+		}
```

Comment:
It seems like this is a bit wonky. What if the user did 'debug=01'? Also this accepts 'debugboogie=1'. So it seems like if we see 'debug' we should just turn on this mode.

Reply:
I just followed the option handling pattern used in the rest of the plugin.

Reply:
I have de-wonkied this in the revised patch.